TrekAddress.html

```diff
@@ -18,128 +18,126 @@
-                          <option>OR</option>
-                          <option>PA</option>
-                          <option>RI</option>
-                          <option>SC</option>
-                          <option>SD</option>
-                          <option>TN</option>
-                          <option>TX</option>
-                          <option>UT</option>
-                          <option>VT</option>
-                          <option>VA</option>
-                          <option>WA</option>
-                          <option>WV</option>
-                          <option>WI</option>
-                          <option>WY</option>
-                        </select>
-                      </div>
-                      <div class="form-group col-md-2">
-                        <label for="inputZip">Zip</label>
-                        <input type="text" class="form-control" id="zip">
-                      </div>
+            <div class="navbar">
+                <div class = "col-md-2">
+                    <nav class="navbar">
+                         <a class=" type='button'" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+                             <img class="logo"  src="SmallTreklogo.png" style = "width:100px;height:100px;">
+                        </a> 
+                    </nav>
+                </div>
+                <div class= "col-md-2">
+                    <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+                        <a class="navbar-brand" href="portfolio.html">Our Philosphy</a>
+                        <a class="navbar-brand" href="index.html">Start Over</a>
                     </div>
-                    <a type="submit" class="btn btn-dark" href="#" id="submitaddress">Submit</a>
-                  </form>
-                  
-               
-            </div>
-        </div>
-        <div class="col-md-"></div>
-    </div>  
-<footer>
-    <nav class="navbar mx-auto fixed-bottom navbar-light bg-transparent" style="width: 200px;";>
-        <a class="navbar-brand" href="#">
-            <div class="footer-text">
-                <a href="" target="_blank" class="fa fa-facebook"></a>
-                        <a href="" target="_blank" class="fa fa-instagram"></a>
-                        <a href="" target="_blank" class="fa fa-linkedin"></a>
+                </div>
+                <div class="col-md-8"></div>
             </div>
-        </a>
-    </nav>
-
-</footer>
+ 
+    <div class="container">
+        <div class="row">
+            <div class="col-md-2"></div>
 
-</div> 
+            <div class="col-md-8">
+                <div class="firstQuestions">
+                    <form>
+                        <h1 class= "AddressHeader">Starting</h1>
+                        <div class="form-group">
+                            <label for="inputAddress">Address</label>
```

Comment:
I changed the label 'for' fields to match the id fields, there will probably a conflict with the pull request I submitted